### DISCUSSION TITLE
How to stop Symfony from sending "PHPSESSID" cookie?

### DISCUSSION THREAD

**Discussion by ghost** [symfony/symfony#43110] [Q&A]
I already made sure stateless: true is set on the routes, nonetheless the PHPSESSID cookie keeps being sent. I couldn't find any resource about how to disable this cookie.

**Comment by michaoj** [comment]
This is not a symfony config but a php one .

This should be done in php.ini or by using ini_set directives

`session.use_cookies = 0`

or 

`ini_set('session.use_cookies', '0');`
